build.sbt

```diff
@@ -128,37 +128,35 @@ lazy val otelExtension = (project in file("otel-extension"))
   )
   .dependsOn(core % "provided->compile;compile->compile", testkit % "it,test")
 
-def exampleCommonSettings = Seq(
-  publish / skip := true,
-  resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots",
-  run / javaOptions ++= Seq(
-    s"-javaagent:$projectRootDir/opentelemetry-javaagent-$OpentelemetryMinor0Version.jar",
-    s"-Dotel.javaagent.extensions=${(otelExtension / assembly).value.absolutePath}",
-    "-Dotel.javaagent.debug=true"
-  ),
-  libraryDependencies ++= {
-    logback ++ Seq(
-      "io.grpc"             % "grpc-netty-shaded" % "1.54.0",
-      "org.wvlet.airframe" %% "airframe-log"      % AirframeVersion
-    )
-  },
-  run / fork         := true,
-  run / connectInput := true
-)
-
-lazy val exampleAkka = (project in file("examples/akka"))
+def exampleProject(project: Project) = project
   .disablePlugins(sbtassembly.AssemblyPlugin)
-  .settings(exampleCommonSettings)
+  .settings(
+    publish / skip := true,
+    resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots",
+    run / javaOptions ++= Seq(
+      s"-javaagent:$projectRootDir/opentelemetry-javaagent-$OpentelemetryMinor0Version.jar",
+      s"-Dotel.javaagent.extensions=${(otelExtension / assembly).value.absolutePath}",
+      "-Dotel.javaagent.debug=true"
+    ),
+    libraryDependencies ++= {
+      logback ++ Seq(
+        "io.grpc"             % "grpc-netty-shaded" % "1.54.0",
+        "org.wvlet.airframe" %% "airframe-log"      % AirframeVersion
+      )
+    },
+    run / fork         := true,
+    run / connectInput := true
+  )
+  .dependsOn(core, testkit % "test")
+
+lazy val exampleAkka = exampleProject(project in file("examples/akka"))
   .settings(
     name := "mesmer-akka-example",
     libraryDependencies ++= {
       akka ++
       scalatest.map(_ % "test") ++
       akkaTestkit.map(_ % "test") ++
-      akkaPersistance ++ Seq(
-        "io.circe"                      %% "circe-core"                        % CirceVersion,
-        "io.circe"                      %% "circe-generic"                     % CirceVersion,
-        "io.circe"                      %% "circe-parser"                      % CirceVersion,
+      akkaPersistance ++ circe ++ Seq(
```

Comment:
persistance -> persistence

Reply:
This change is not a part of this PR, but sure, I can fix it on the way.